controllers/gateway_controller.go

```diff
@@ -258,21 +256,15 @@ func (r *GatewayReconciler) cleanupGatewayResources(ctx context.Context, gw *gat
 
 func (r *GatewayReconciler) updateGatewayStatus(ctx context.Context, serviceNetworkStatus *latticestore.ServiceNetwork, gw *gateway_api.Gateway) error {
 
-	glog.V(6).Infof("updateGatewayStatus: updating last transition time \n")
```

Comment:
can we leave this for debugging?

Reply:
updateGatewayStatus is only called in a single place and there is a log beforehand, so it is a duplicate IMO